Support for explicit New Relic revision (#148)
fabfuel/ecs-deploy: README.rst

    $ export NEW_RELIC_APP_ID=1234567890
    $ ecs deploy my-cluster my-service

Optionally you can provide additional information for the deployment:

- ``--comment "New feature X"`` - comment to the deployment
- ``--user john.doe`` - the name of the user who deployed with
- ``--newrelic-revision 1.0.0`` - explicitly set the revison to use for the deployment

> Note: If one of ``--tag`` or ``--newrelic-revision`` is not provided, the deployment will not be recorded.

Reviewer: I think this sounds like both are required. Maybe this?
```suggestion
Note: If neither ``--tag`` nor ``--newrelic-revision`` are provided, the deployment will not be recorded.
```
